torch/autograd/profiler.py

```diff
@@ -1134,29 +1135,29 @@ def rewrite_name(name, with_wildcard=False):
 # Parsing of kineto profiler events
 def parse_kineto_results(result):
     # result.events() has most of the events - PyTorch op-level and device-level events
-    # result.legacy_events() has events not yet ported to kineto
-    # (e.g. start/stop marks, tensor memory allocator events)
 
-    # First, find __start_profile mark to get the absolute time of the start of the trace;
-    # save memory allocation records
-    start_record = None
-    mem_records = []
-    for record in itertools.chain(*result.legacy_events()):
-        if record.kind() == 'mark' and record.name() == '__start_profile':
-            assert start_record is None
-            start_record = record
-        if record.kind() == 'memory_alloc':
-            mem_records.append([record, False])
-    assert start_record is not None, "Invalid profiler output, __start_profile is missing"
+    trace_start_us = result.trace_start_us()
+    mem_records = [[evt, False] for evt in result.events() if evt.name() == "[memory]"]
```

Comment:
What does 'False' mean in `[evt, False]`?

Reply:
it's a flag that is used later in the code to filter out "top-level" memory events (events that did not happen in the context of another op), that are outputted separately